ar_exercise/exercise.rb

```diff
@@ -23,26 +23,41 @@ class Apartment < ActiveRecord::Base
 ################################################
 
 
-################################################
+# ###############################################
 # FINDING / SELECTING
-################################################
+# ###############################################
 
 # Hint, the following methods will help: `where`, `all`, `find`, `find_by`
 
 # Use Active record to do the following, and store the results **in a variable**
 # example: get every tenant in the DB
 all_tenants = Tenant.all
-
+binding.pry
 # get the first tenant in the DB
+first_tenant = Tenant.first
+binding.pry
 # get all tenants older than 65
+elders = Tenant.where("age > ?", "65")
+
 # get all apartments whose price is greater than $2300
+expensive = Tenant.where("monthly_rent > ?", "2300")
 # get the apartment with the address "6005 Damien Corners"
+damien_corners = Apartment.find_by(address: "6005 Damien Corners")
 # get all tenants in that apartment
-
+damien_tenants = Tenant.all.where(apartment: damien_corners)
```

Comment:
becasue you set up `has_many` and `belongs_to` these queries become much easier. Yours works, but so will this:

```ruby
damien_tenants = damien_corners.tenants
```